docs/en/advanced/Client-Logging.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: global
+title: Client Logging
+nickname: Client Logging
+group: Advanced
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+## Introduction
+
+This page summarizes how to find client-specific logs and modify logging levels in order to debug
+client-sided issues with your Alluxio installation.
+
+## Log Location
+
+Depending on which compute framework you are using, the location of the logs can vary. We
+recommend looking at the specific documentation for your compute framework to determine where to
+find alluxio cient logs.
+
+Here are links to compute frameworks with information where logs may be found and how to configure
+
+- [Apache Spark]({{ site.baseurl }}{% link en/compute/Compute-Spark.md %})
+- [Apache Hadoop]({{ site.baseurl }}{% link en/compute/Compute-Hadoop-MapReduce.md %})
+- [Apache HBase]({{ site.baseurl }}{% link en/compute/Compute-HBase.md %})
+- [Apache Hive]({{ site.baseurl }}{% link en/compute/Compute-Hive.md %})
+- [Presto]({{ site.baseurl }}{% link en/compute/Compute-Presto.md %})
+
+However, by setting the `alluxio.logs.dir` property you can modify where the logs are stored.
+This Alluxio property is a JVM system property which cannot be set from an
+`alluxio-site.properties` file. For more information on setting JVM system properties see
+the [configuration documentation]({{site.baseurl}}{% link en/advanced/Configuration-Settings.md %}#configuration-sources)
+
+## Log Levels
+
+Often it's useful to change the logLevel of the Alluxio client running in the compute framework
+(e.g. Spark, Presto) process, and save it to a file for debugging. To achieve this, you can pass
+the following Java options to the compute framework process.
+
+For example, the options `-Dalluxio.logs.dir=/var/alluxio/ -Dalluxio.logger.type=USER_LOGGER
+-Dlog4j.configuration=/tmp/alluxio/conf/log4j.properties` will instruct Alluxio client to use the
```

Comment:
does this work if the compute framework already has a `log4j.properties` file on its classpath?

Reply:
I think it will probably be better to just instruct them to use the framework-specific logging documentation?

Reply:
👍